### PR TITLE
Translation descriptions CV2-3760

### PR DESCRIPTION
**Pull request by brianfleming**
## Description

This should be the final batch of missing content translation descriptions

References: [CV2-3760](https://meedan.atlassian.net/browse/CV2-3760)

## Type of change

- [x] Performance improvement and/or refactoring (non-breaking change that keeps existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Security mitigation or enhancement
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Automated test (add or update automated tests)

## How has this been tested?

manual testing

## Things to pay attention to during code review

Should just be content, a few inline comments for engineering assistance

## Checklist

- [x] I have performed a self-review of my own code
- [x] I've made sure my branch is runnable and given good testing steps in the PR description
- [x] I considered secure coding practices when writing this code. Any security concerns are noted above.
- [x] I have commented my code in hard-to-understand areas, if any
- [ ] I have made needed changes to the README
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I implemented any new components, they are self-contained, their `propTypes` are declared and they use React Hooks and, if data-fetching is required, they use Relay Modern with fragment containers
- [x] I have removed the /* eslint-disable @calm/react-intl/missing-attribute */ from any files I've worked on and added a `description` prop to all `<FormattedMessage />` components that were missing it.
- [x] To the best of my knowledge, any new styles are applied according to the design system
- [ ] If I added a new external dependency, I included a rationale for doing so and an estimate of the change in bundle size (e.g., checked in https://bundlephobia.com/)

[CV2-3760]: https://meedan.atlassian.net/browse/CV2-3760?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/components/MappedMessage.js [hunk 1]
```diff
@@ -17,7 +16,7 @@ function mapGlobalMessage(intl, msgKey, values) {
 
 function FormattedGlobalMessage({ messageKey, ...rest }) {
   const message = globalStrings[messageKey];
-  return <FormattedMessage {...message} {...rest} />;
+  return <FormattedMessage {...message} {...rest} />; // eslint-disable-line @calm/react-intl/missing-attribute
```
Comment (brianfleming) [inline src/app/components/MappedMessage.js:19]: Note to reviewers: I am not sure how to resolve this, or if its necessary to. Any guidance would be appreciated

File: src/app/components/team/SmoochBot/SmoochBotTextEditor.js [hunk 2]
```diff
@@ -15,7 +14,7 @@ const useStyles = makeStyles(theme => ({
 
 const ValueOrPlaceholder = ({ value, field, children }) => (value || !placeholders[field])
   ? children(value)
-  : <FormattedMessage {...placeholders[field]}>{children}</FormattedMessage>;
+  : <FormattedMessage {...placeholders[field]}>{children}</FormattedMessage>; // eslint-disable-line @calm/react-intl/missing-attribute
```
Comment (brianfleming) [inline src/app/components/team/SmoochBot/SmoochBotTextEditor.js:17]: Note to reviewers: I am not sure how to resolve this, or if its necessary to. Any guidance would be appreciated

File: src/app/components/user/UserUtil.js [hunk 3]
```diff
@@ -23,7 +22,7 @@ const messages = defineMessages({
 });
 
 const LocalizedRole = ({ role, children }) => role ?
-  <FormattedMessage {...messages[role]}>{children}</FormattedMessage>
+  <FormattedMessage {...messages[role]}>{children}</FormattedMessage> // eslint-disable-line @calm/react-intl/missing-attribute
```
Comment (brianfleming) [inline src/app/components/user/UserUtil.js:25]: Note to reviewers: I am not sure how to resolve this, or if its necessary to. Any guidance would be appreciated

**Comment by brianfleming** [comment]
There are 3 instances left of `eslint-disable-line @calm/react-intl/missing-attribute` because of the spread operator linking back to the `MappedMessage.js` file.
I noted those places in inline comments above. I am not sure if this is worth addressing, but any engineering insight would be helpful.

**Comment by brianfleming** [comment]
Re: the last 3 lint disable statements. Since this PR is just all content, we could also just merge and address the last 3 at a later date.

**Review by amoedoamorim — APPROVED** [review]
This is gold! Thank you Brian!

**Comment by amoedoamorim** [comment]
> Re: the last 3 lint disable statements. Since this PR is just all content, we could also just merge and address the last 3 at a later date.

I don't have an immediate answer to how to fix those, need to go back and figure out how that's being use and what are the implications there. Let's not worry about that for now and take care of that on some other time :100: 

That's pretty contained and the real value here is that we've now free of file-level `eslint-disable @calm/react-intl/missing-attribute` :heart_eyes:  Thank you!

**Pull request merged**